src/qrcode/utils.ts

```diff
@@ -126,10 +127,17 @@ export function* handleSecureSend(
 }
 
 function* extractQRAddressData(qrCode: QrCode) {
-  // Regex matches any 40 hexadecimal characters prefixed with "0x" (case insensitive)
-  if (/^0x[a-f0-9]{40}$/gi.test(qrCode.data)) {
+  // Check if the QR code is a valid address, does not perform checksum validation
+  if (isAddress(qrCode.data, { strict: false })) {
```

Comment:
what's this `strict` option?  I don't see it in the docs...

Reply:
https://github.com/wevm/viem/blob/main/src/utils/address/isAddress.ts#L12